Unable to clone Memtest86+ from Git

I run in to the following error when building a new release version. 

Steps 

1.     `git clone https://github.com/pcengines/pce-fw-builder.git -b 1.52.5`
2.      `cd pce-fw-builder`
3.     `./build.sh release v4.14.0.4 apu2`

Results in the following error. 

```  Compile checking out/src/fw/xen.o
  Compile checking out/src/fw/acpi.o
  Compile checking out/src/fw/mptable.o
  Compile checking out/src/fw/pirtable.o
  Compile checking out/src/fw/smbios.o
  Compile checking out/src/fw/romfile_loader.o
  Compile checking out/src/fw/dsdt_parser.o
  Compile checking out/src/fw/vpd.o
  Compile checking out/src/fw/lib_vpd.o
  Compile checking out/src/hw/virtio-ring.o
  Compile checking out/src/hw/virtio-pci.o
  Compile checking out/src/hw/virtio-mmio.o
  Compile checking out/src/hw/virtio-blk.o
  Compile checking out/src/hw/virtio-scsi.o
  Compile checking out/src/hw/tpm_drivers.o
  Compile checking out/src/hw/nvme.o
  Compiling whole program out/ccode32flat.o
  Compiling whole program out/code32seg.o
  Compiling whole program out/ccode16.o
  Compiling to assembler out/src/asm-offsets.s
  Generating offset file out/asm-offsets.h
  Compiling (16bit) out/romlayout.o
  Building ld scripts
Version: rel-1.14.0.1-0-g8610266a
Fixed space: 0xe05b-0x10000  total: 8101  slack: 3  Percent slack: 0.0%
16bit size:           26688
32bit segmented size: 1292
32bit flat size:      21700
32bit flat init size: 36896
Lowmem size:          2240
f-segment var size:   1392
  Linking out/rom16.o
  Stripping out/rom16.noexec.o
  Linking out/rom32seg.o
  Stripping out/rom32seg.noexec.o
  Linking out/rom.o
  Prepping out/bios.bin.prep
Total size: 92384  Fixed: 51072  Free: 38688 (used 70.5% of 128KiB rom)
  Creating out/bios.bin.elf
    Cloning Memtest86+ from Git
Cloning into '/home/coreboot/coreboot/payloads/external/Memtest86Plus/memtest86plus'...
fatal: unable to access 'https://review.coreboot.org/memtest86plus.git/': server certificate verification failed. CAfile: /etc/ssl/certs/ca-certificates.crt CRLfile: none
make[1]: *** [Makefile:17: /home/coreboot/coreboot/payloads/external/Memtest86Plus/memtest86plus] Error 128
make: *** [payloads/external/Makefile.inc:281: payloads/external/Memtest86Plus/memtest86plus/memtest] Error 2
cp: cannot stat 'coreboot/build/coreboot.rom': No such file or directory
sha256sum: apu2_v4.14.0.4.rom: No such file or directory```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM coreboot/coreboot-sdk:1.52
+FROM coreboot/coreboot-sdk:2021-04-06_7014f8258e
 MAINTAINER Piotr Król <piotr.krol@3mdeb.com>
 USER root
 COPY ./scripts/entrypoint.sh /entrypoint.sh

